hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/diskUsage/diskUsage.tsx

```diff
@@ -409,30 +422,44 @@ export class DiskUsage extends React.Component<Record<string, object>, IDUState>
                 {(duResponse.size > 0) ?
                   ((duResponse.size > 0 && duResponse.subPathCount === 0) ?
                     <div style={{height: 800}}>
+                    <div  className='metadatainformation'>
                       <br/> {' '}
-                      <h3>This object is a key with a file size of {this.byteToSize(duResponse.size, 1)}.<br/> {' '}
                         You can also view its metadata details by clicking the top right button.
-                      </h3>
-                    </div> :
+                    </div>
+                    <Plot
+                    data={plotData}
+                    layout={
+                      {
+                        width: 800,
+                        height: 750,
+                        font: {
+                          family: 'Roboto, sans-serif',
+                          size: 15
+                        },
+                        showlegend: true,
+                        title: 'Disk Usage for ' + returnPath + ' (Total Size: ' + this.byteToSize(duResponse.size, 1) + ')'
+                      }
+                    }/>
+                  </div>
+                    :
```

Comment:
I tried it and we could remove the second condition inside this nested condition. The only difference between these two `Plot`'s is the `You can also view its metadata details by clicking the top right button.` div, which can be removed and the `onClick`. We can add the condition inside the `onClick`, like this: `onClick={(duResponse.subPathCount === 0) ? undefined : e => this.clickPieSection(e, returnPath)}`
I also removed the `duResponse.size > 0` from the condition, as that is checked previously.

Reply:
There are two condition If  (duResponse.size > 0 && duResponse.subPathCount === 0)  as per siyao need to show empty Pie chart  no further clicks on this pie chart. If condition is false show regular pie chart as suggested by siyao.

Reply:
The `duResponse.size > 0` check in the second condition is unnecessary, as we check it in the line above it in the first condition. That's why we can keep only `duResponse.subPathCount === 0` in the second condition. We can remove the Plot you added and add a change in the already existing Plot like this: `{(duResponse.subPathCount === 0) ? undefined : e => this.clickPieSection(e, returnPath)}` (and with this change we remove the "You can also view its metadata details by clicking the top right button." from the Plot as well, if we already planned to fix that)